scripts/publish-to-npm.js

```diff
@@ -18,38 +18,50 @@ const main = () => {
 
     const branchName = sh.exec('git branch --show-current', {silent: true}).trim()
     // DEBUG
+    // const branchName = 'next'
     // const branchName = 'release-3.0.x'
-    // const branchName = 'release-retail-react-app-1.0.x'
+    // const branchName = 'release-commerce-sdk-react-10.0.x'
 
     console.log('--- Given the current branch:', branchName)
 
     const isNightly = branchName === 'nightly-releases'
+    const isNextBranch = branchName === 'next'
 
     if (isNightly) {
         console.log('--- Nightly release detected. Releasing all packages...')
-        publishPackages([], true)
-    } else {
-        const matched = branchName.match(RELEASE_ONE_PACKAGE)
-        const packageName = matched && matched[1]
-
-        if (packageName) {
-            console.log(`--- Releasing ${packageName}...`)
-            publishPackages([packageName])
-        } else {
-            console.log('--- Releasing all packages...')
-            publishPackages()
-        }
+        publishPackages({isNightly: true})
+        return
     }
+
+    if (isNextBranch) {
+        console.log('--- The `next` branch detected. Releasing all packages with `next` tag...')
+        publishPackages({npmTag: 'next'})
+        return
+    }
```

Comment:
Detecting the `next` branch, while also doing a little bit of code cleanup/refactoring